helper/cuckootable.c

```diff
@@ -285,7 +289,7 @@ odph_cuckoo_table_create(
 	queue = odp_queue_create(queue_name, &qparam);
 	if (queue == ODP_QUEUE_INVALID) {
 		ODPH_DBG("failed to create free_slots queue\n");
-		odp_pool_destroy(pool);
+		(void) odp_pool_destroy(pool);
```

Comment:
This line has a checkpatch warning:
CHECK: No space is necessary after a cast
#38: FILE: helper/cuckootable.c:292:
+		(void) odp_pool_destroy(pool);


You can run checkpatch locally e.g. like this:
git format-patch -o ../outgoing/ -6
perl scripts/checkpatch.pl ../outgoing/*

Travis also runs it as the last test case. However, Travis status ignores failure in checkpatch case.